docs/cephfs-snap-clone.md

```diff
@@ -0,0 +1,78 @@
+# Create CephFS snapshot and Clone Volume
+
+- [Create CephFS snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
+  - [Prerequisite](#prerequisite)
+  - [Snapshot](#snapshot)
+    - [Create snapshotclass](#create-snapshotclass)
+    - [Create snapshot](#create-snapshot)
+  - [Restore snapshot to a new PVC](#restore-snapshot-to-a-new-pvc)
+  - [Clone PVC](#clone-pvc)
+
+## Prerequisite
+
+- For snapshot functionality to be supported for your Kubernetes cluster, the
+  Kubernetes version running in your cluster should be >= v1.17. We also need the
+  `snapshot controller` deployed in your Kubernetes cluster along with `csi-snapshotter`
+  sidecar container.
+  Refer [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/)
+  for more information on these sidecar controllers. There should
+  be a `volumesnapshotclass` object present in the cluster
+  for snapshot request to be satisified.
+
+## Snapshot
+
+### Create snapshotclass
+
+```console
+kubectl create -f ../examples/cephfs/snapshotclass.yaml
+```
+
+### Create snapshot
+
+The snapshot is created on/for an existing PVC. You should
+have a PVC in bound state before creating snapshot from it.
+please refer pvc creation [doc](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-cephfs.md)
+for more information on how to create a PVC.
+
+```console
+$ kubectl create -f ../examples/cephfs/snapshot.yaml
+volumesnapshot.snapshot.storage.k8s.io/cephfs-pvc-snapshot created
+```
+
+- Get details about the snapshot
+
+```console
+$ kubectl get volumesnapshot
+NAME                  READYTOUSE   SOURCEPVC    SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS                SNAPSHOTCONTENT                                    CREATIONTIME   AGE
```

Comment:
this alignment looks to be broken https://github.com/ceph/ceph-csi/blob/5e305ab65cb51f0cc345ddaa0ec449dc0d816b1f/docs/cephfs-snap-clone.md#create-snapshot

Reply:
@Madhu-1 can you refresh https://github.com/ceph/ceph-csi/pull/1343/commits/5e305ab65cb51f0cc345ddaa0ec449dc0d816b1f?short_path=9b1252c#diff-9b1252c8f53ed336e48e29788d07810f ?

Reply:
I see.. one sec

Reply:
@Madhu-1 should be good now